x/datadeal/types/message_deal_test.go

```diff
@@ -46,7 +46,7 @@ func TestMsgCreateDealValidateBasicEmptyValue(t *testing.T) {
 
 	err := msg.ValidateBasic()
 	require.ErrorIs(t, err, sdkerrors.ErrInvalidRequest)
```

Comment:
It would be nice to add a test where both data_schema and pd exist.

Reply:
Ok. I'll cover it as well.

Reply:
done in [6554699](https://github.com/medibloc/panacea-core/pull/632/commits/65546994e9cbe071c707dcf94fb98a332bd36ae8)